sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/ChangeFeedTest.java

```diff
@@ -546,4 +548,10 @@ private static void waitAtleastASecond(Instant befTime) throws InterruptedExcept
             Thread.sleep(100);
         }
     }
+
+    @Retention(java.lang.annotation.RetentionPolicy.RUNTIME)
+    @Target({METHOD})
+    @interface Tag {
```

Comment:
Where is this used ?

Reply:
in the changeFeedTest.java

Reply:
@Test(groups = { "emulator" }, enabled = false, timeOut = TIMEOUT)
    @Tag(name = "EnableFullFidelity")
    public void changeFeed_fullFidelity_fromNow() throws Exception {
        changeFeed_withUpdatesAndDelete(true);
    }